### PR TITLE
Simplify Riak cluster creation HostAndPort was added

### PR DESCRIPTION
**Pull request by srgg**

### CODE REVIEW

File: src/main/java/com/basho/riak/client/core/util/HostAndPort.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+import java.io.Serializable;
+import java.net.InetSocketAddress;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * An immutable representation of a host and port.
+ *
+ * This Own implementation is created to reduce the amount of 3rd party libraries
```
Comment (alexmoore) [inline src/main/java/com/basho/riak/client/core/util/HostAndPort.java]: What's "Own" ?

File: src/main/java/com/basho/riak/client/core/util/HostAndPort.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    private final int port;
+    private final String host;
+    private transient InetSocketAddress inetAddress;
+
+    private HostAndPort(String host, int port){
+        this.host = host;
+        this.port = port;
+    }
+
+    public boolean hasPort() {
```
Comment (alexmoore) [inline src/main/java/com/basho/riak/client/core/util/HostAndPort.java]: Let's move the brackets to the next line style here. 

```
if (blah)
{...
```

**Comment by alexmoore** [comment]
Looks mostly good, just need to fix the formatting of the brackets to match the project.  +0.75 otherwise.

**Comment by srgg** [comment]
@alexmoore I also realized that introduced approach should be propagated to the RiakClient level as a newClient() routine.. Will add it and adjust PR according your comments.

**Pull request merged**